cluster-autoscaler/core/scaledown/actuation/group_deletion_scheduler.go

```diff
@@ -45,17 +46,21 @@ type GroupDeletionScheduler struct {
 	nodeDeletionTracker *deletiontracker.NodeDeletionTracker
 	nodeDeletionBatcher batcher
 	evictor             Evictor
+	deleteOptions       options.NodeDeleteOptions
+	drainabilityRules   rules.Rules
 	nodeQueue           map[string][]*apiv1.Node
 	failuresForGroup    map[string]bool
 }
 
 // NewGroupDeletionScheduler creates an instance of GroupDeletionScheduler.
-func NewGroupDeletionScheduler(ctx *context.AutoscalingContext, ndt *deletiontracker.NodeDeletionTracker, b batcher, evictor Evictor) *GroupDeletionScheduler {
+func NewGroupDeletionScheduler(ctx *context.AutoscalingContext, ndt *deletiontracker.NodeDeletionTracker, b batcher, evictor Evictor, deleteOptions options.NodeDeleteOptions, drainabilityRules rules.Rules) *GroupDeletionScheduler {
```

Comment:
We don't need these parameters anymore, right? Could you clean them up?

Reply:
Done